### DISCUSSION TITLE
Send frequent reminder for PR open to review

### DISCUSSION THREAD

**Discussion by Ankit-Laddha** [cakarci/pull-request-workflow#40] [Ideas]
WHAT?
- Add ability to send frequent (configurable timeline) slack reminder in Team Channel for PR open to review
 e.g. every 3 hrs OR twice in a day

WHY?
- Sometimes as a developer, I'm too focused on my current task that I forget to review my team member PRs 💔.
This
  a. deprives my team member for quick feedback on their PR
  b. slows them down to merge their changes => this impacts time to complete the user-story => velocity of the team goes down
  c. have negative impact on developer experience

VALUE?
- Faster reviews of PRs
- Improve developer experience
- Reduces time to close story & Increases velocity of the team

**Comment by cakarci** [comment]
Hi @Ankit-Laddha this feature is introduced in the release https://github.com/cakarci/pull-request-workflow/releases/tag/v1.11.4 🎉 
Thank you for contributing with your ideas 💯
